OxfordBodleian/Juel-Jensen/BDLaethg36.xml

```diff
@@ -102,90 +95,304 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                </seg>
+                        </handNote>
+                    </handDesc>
+                    
+                    <decoDesc>
+                        <decoNote type="ornamentation" xml:id="d1">
+                            <locus target="#40r"></locus>
+<!--                            <desc> Jacopo
+                            </desc>-->
+                        </decoNote>
+                    </decoDesc>
+                    
+                    <additions>
+                        <list>
+                            <item xml:id="e3">
+                                Omitted letters and words have been added, most often interlineally (e.g. <locus target="#23r #28v #45r"/>),
+                                but once in the upper margin with a <foreign xml:lang="gez">tamallas</foreign> sign in the text 
+                                (<locus target="#45r"/>).
+                                There are different type of deletions: erasures (e.g. <locus target="#1r #5v #38v"/>), 
+                                erasures with subsequent addition of line fillers 
+                                (e.g. <locus target="#13r #25v"/>),
+                                marking with over- and underlining (e.g. <locus target="#4r #20r #32r"/>),
+                                marking with over- and underlining with subsequent erasure (e.g. <locus target="#20v #28r #30r"/>),
+                                blotting out (e.g. <locus target="#12r #13r #25v"/>).
+                            </item>
+                        </list>
+                    </additions>
+                    
+                </physDesc>
+
+                    </msPart>
 
-                    <additional>
-                        <adminInfo>
-                            <recordHist>
-                                <source>
-                                  <!--  <listBibl type="secondary">
-                                        <bibl></bibl>
-                                    </listBibl>-->
-                                </source>
-                            </recordHist>
-                        </adminInfo>
-                    </additional>
+<msPart xml:id="p2">
+    <msIdentifier></msIdentifier>
+    
+    <msContents>
+        <summary/>
+        
+        <msItem xml:id="p2_i1">
+            <locus target="#53ra"/>
+            <title type="incomplete">Beginning of a list of the commemoration dates of the apostles</title>
+            <note>The text is reproduced in full below.</note>
+            <incipit xml:lang="gez">
+            ሠረቀ፡ መስከረም፡ በ፩በርተሎሜዎስ፡ ሠረቀ፡ ጥቅምት፡ በ፲ወ፪ማቴዎስ፡ በ፳ወ፪ሉቃስ፡ ሠ፡ ኅ
+            </incipit>
+        </msItem>
+    </msContents>
+    
+    <physDesc>
+        <objectDesc form="Leaf">
+            
+            <supportDesc>
+                <support>
+                    <material key="parchment"/>
+                </support>
+            </supportDesc>
```

Comment:
please add the element extent with information on the amount of leaves and location within the manuscript
```
     <extent>
    <measure unit="leaf" quantity="1">1</measure>
                                    <locus target="#53"/>
```